src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py

```diff
@@ -4738,6 +4744,71 @@ def get_aks_custom_headers(self) -> Dict[str, str]:
         # this parameter does not need validation
         return aks_custom_headers
 
+    def _get_enable_azure_monitor_metrics(self, enable_validation: bool = False) -> bool:
+        """Internal function to obtain the value of enable_azure_monitor_metrics.
+        This function supports the option of enable_validation.
+        When enabled, if both enable_azure_monitor_metrics and disable_azure_monitor_metrics are
+        specified, raise a MutuallyExclusiveArgumentError.
+
+        :return: bool
+        """
+        # print("_get_enable_azure_monitor_metrics being called...")
+        # Read the original value passed by the command.
+        enable_azure_monitor_metrics = self.raw_param.get("enable_azuremonitormetrics")
+        # In create mode, try to read the property value corresponding to the parameter from the `mc` object.
+        if self.decorator_mode == DecoratorMode.CREATE:
+            if (
+                self.mc and
+                self.mc.azure_monitor_profile and
```

Comment:
```suggestion
                hasattr(self.mc, "azure_monitor_profile") and
                self.mc.azure_monitor_profile and
```
may use something like this as a workaround to pass the test cases for old API version